### PR TITLE
Fix/con 24/top block styles

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/CON-24

Added a border: solid 1 px #E5E4E4.
Switched the cursor to a pointer while it hovers the collapser: class="top-block-preview-collapser"

How to test:
- go to Tests page
- go to Test Preview
- check border and cursor on the bottom part
image

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/js/previewer/component/topBlock/scss/topBlock.scss [hunk 1]
```diff
@@ -40,4 +42,15 @@
     display: flex;
     align-items: center;
     justify-content: center;
+    cursor: pointer;
+    border-left: solid 1px #E5E4E4;
```
Comment (andrey-ponamarev) [inline views/js/previewer/component/topBlock/scss/topBlock.scss:46]: Should we create a variable for it?

**Review by andrey-ponamarev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by adriloma — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] bumping versions are applied (if applicable)

**Pull request merged**